src/packaging/appdepsdk/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "Microsoft.DotNet.AppDep": "1.0.6-prerelease-00003"
```

Comment:
We can use `00004`. It has a fix with the stubbed out `System.Resources.ResourceManager`. I didn't get around to actually updating it and your changes make that super easy.